mesh/mesh.cpp

```diff
@@ -4411,6 +4411,49 @@ Mesh::Mesh(real_t *vertices_, int num_vertices,
    FinalizeTopology();
 }
 
+Mesh::Mesh( const NURBSExtension& ext )
+ : attribute_sets(attributes), bdr_attribute_sets(bdr_attributes)
+{
+   SetEmpty();
```

Comment:
@david-kamensky can you respond to ☝️ ?

Reply:
@YohannDudouit I think the above two comments from me and Tzanio crossed paths with each other, since I'd inadvertently left mine as "Pending" for a while.